### DISCUSSION TITLE
github support

### DISCUSSION THREAD

**Discussion by AFCC** [community/community#31692] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, my name is Connor and I work for the company Outseer security. Our company is a company with a wide range of clients from all over the world. We provide them with internet security services on the wide web in all matters of copyright infringement, trademark, phishing, trojans and more. We, on behalf of our customers, try to solve these problems for them by talking to the parties responsible for the same platform where the problem is. The fact that there are many situations of copyright infringement on the platform of Github. We fill out tickets and send them to the support team but don't always get a reply or see that the content has been removed. I would love to know if there is an easier way to talk to Github or we can remove content that violates the copyright of our customers. I would appreciate any help or advice

**Comment by radar-blip** [comment, marked as answer]
Perhaps a better way to contact support: [Github Support](https://support.github.com/request)

**Reply by nathaneneja** [reply]
yeah

**Comment by nathaneneja** [comment]
do you know that i am a master at coding
